### PR TITLE
updated english translation

### PR DESCRIPTION
**Pull request by agderes**

### CODE REVIEW

File: src/locales/gb.json [hunk 1]
```diff
@@ -68,5 +68,30 @@
...
+            "Be democratic, giving full opportunity in decision making.",
+            "Be accountable to the mission entrusted to you by the campus leadership as part of the IEP.",
+            "Be innovative before the new challenges of Colombian Education.",
+            "Be persistent in your work, observing day by day the educational process in each student. Providing solutions to the difficulties that may arise",
+            "Know and be sensitive to the situation that each family is experiencing.",
+            "Be aware and knowledgeable about the reality of the country."
+        ]
+      }
 }
+
```
Comment (1kh4l) [inline src/locales/gb.json:97]: This space is unnecessary

File: src/locales/gb.json [hunk 2]
```diff
@@ -68,5 +68,30 @@
             "Currently, in 2014, the Congregation of Basilian Priests is organizing the House for the Novitiate in the city of Santa Martha, as a budding experience, it will surely require immense efforts, but, redundant in the future of fruits for the work of the Kingdom here on Earth, among our People. "
         ],
         "our-work": "From our experience and Basilian charism carry out a new evangelization that fosters life in all its senses promoting a Community in Christ that nurtures and celebrates the liberation, organization, development and participation of people to live here fully in brotherhood the Kingdom of God, now and forever."
-    }
+    },
+    "content-education": {
+        "students": [
+            "The INSA wants the student.",
```
Comment (1kh4l) [inline src/locales/gb.json:74]: wants the student to

File: src/locales/gb.json [hunk 3]
```diff
@@ -68,5 +68,30 @@
...
+            "Get to know and value yourself.",
```
Comment (1kh4l) [inline src/locales/gb.json:75]: wouldn't be himself?

File: src/locales/gb.json [hunk 4]
```diff
@@ -68,5 +68,30 @@
...
+            "Value and enjoy your family and provide solutions to family conflicts.",
+            "Be critical and sensitive to the reality around you.",
+            "Develop your personal potential and skills that allow you to insert yourself in the productive sector.",
+            "Be able to live in a community and work in a team.",
```
Comment (1kh4l) [inline src/locales/gb.json:79]: work as a team? don't you think?

Comment (agderes) [reply src/locales/gb.json:79]: I think it is correct.  work in a team... but you are native so do I need to change it as "work as a team"

Comment (1kh4l) [reply src/locales/gb.json:79]: lol, we're gonna have someone who review Englihs as well so no worries, was just wondering

**Pull request merged**